components/server/src/user/user-controller.ts

```diff
@@ -162,7 +162,7 @@ export class UserController {
                 });
 
                 // Simply redirect to the app for now
-                res.redirect("/", 307);
+                res.redirect("/teams/new", 307);
```

Comment:
We could check whether there already is a team, and only do it when there are 0 teams or so...? :thinking: 

Anyway, already improves the flow. :heavy_check_mark: